code/modules/antagonists/cult/blood_magic.dm

```diff
@@ -418,43 +418,28 @@
 	color = RUNE_COLOR_RED
 	invocation = "Fuu ma'jin!"
 
-/obj/item/melee/blood_magic/stun/afterattack(atom/target, mob/living/carbon/user, proximity)
+/obj/item/melee/blood_magic/stun/afterattack(mob/living/target, mob/living/carbon/user, proximity)
```

Comment:
Cast to a living into a proc instead of changing the parameters here - what it did prior was fine (albeit needs a better var name)

Reply:
There is already an early return to check if `target` is living.  Nowhere in this proc do we use it for objects or things that aren't alive otherwise we would use `atom/target`

Reply:
Fair enough